### PR TITLE
Cleanup and warnings

### PR DESCRIPTION
**Pull request by PipGrylls**
Mostly just cleaning things to remove warnings and a bit of refactoring around dependencies

### CODE REVIEW

File: fixtures/demo.json [hunk 1]
```diff
@@ -1,714 +1,4 @@
 [
```
Comment (jag1g13) [inline fixtures/demo.json:1]: Has this file been kept deliberately as well as the moved one?

Comment (PipGrylls) [reply fixtures/demo.json:1]: I didn't want to delete it as the documentation still references it https://softwaresaved.github.io/lowfat/doc/testing.html
I will update the documentation and we can delete the old?

Comment (PipGrylls) [reply fixtures/demo.json]: I think the following commits fix that?

**Comment by PipGrylls** [comment]
@jag1g13 Are you happy to merge this now?

File: fixtures/demo.json (hunk 1 above)
Comment (jag1g13) [reply fixtures/demo.json]: Nearly! The other deleted files (flatpages and sites) we actually do need to keep - those are part of the deployment process of a production system - but moving them into the same place as you did the demo fixtures would make sense

Comment (PipGrylls) [reply fixtures/demo.json]: Sorry, that was my ignore file and not paying enough attention they should be back now!

**Pull request merged**